addons/mail/static/src/activity/activity_menu.xml

```diff
@@ -8,29 +8,34 @@
             <span t-if="activity.counter" class="badge"><t t-esc="activity.counter"/></span>
         </t>
         <t t-set-slot="default">
-            <div class="o-mail-activity-menu d-flex flex-column overflow-auto cursor-pointer">
-                <t t-foreach="activity.activities" t-as="group" t-key="group.id">
-                    <div class="o-mail-activity d-flex p-2 border-bottom" t-on-click="() => this.openActivityGroup(group)">
-                        <img class="o-mail-activity-img" alt="Activity" t-att-src="group.icon"/>
-                        <div class="flex-grow-1">
-                            <div class="d-flex px-2">
-                                <span class="flex-grow-1"><t t-esc="group.name"/></span>
-                                <t t-foreach="group.actions" t-as="action" t-key="action_index">
-                                    <button class="btn fa p-0" type="button" t-att-title="action.name" t-att-class="action.icon"/>
-                                </t>
-                            </div>
-                            <div t-if="group.type === 'activity'" class="d-flex">
-                                <button t-if="group.overdue_count" type="button" class="btn btn-link py-0 px-2" t-on-click.stop="() => this.openActivityGroup(group, 'overdue')"><t t-esc="group.overdue_count"/> Late</button>
-                                <span t-else="" class="text-muted py-0 px-2">0 Late</span>
-                                <button t-if="group.today_count" type="button" class="btn btn-link py-0 px-2" t-on-click.stop="() => this.openActivityGroup(group, 'today')"><t t-esc="group.today_count"/> Today</button>
-                                <span t-else="" class="text-muted py-0 px-2">0 Today</span>
-                                <span class="flex-grow-1"/>
-                                <button t-if="group.planned_count" type="button" class="btn btn-link py-0 px-2" t-on-click.stop="() => this.openActivityGroup(group, 'upcoming_all')"><t t-esc="group.planned_count"/> Future</button>
-                                <span t-else="" class="text-muted py-0 px-2">0 Future</span>
+            <div class="o-mail-activity-menu">
+                <div t-if="activity.counter === 0" class="o-mail-no-activity dropdown-item-text text-center d-flex justify-content-center">
```

Comment:
`text-center` is useless with `d-flex align-items-center`, as you use flexbox alignment instead.